gradle.properties

```diff
@@ -3,26 +3,26 @@ group=io.ballerina
 version=0.9.0-beta.2-SNAPSHOT
 
 #dependency
-ballerinaLangVersion=2.0.0-beta.1
+ballerinaLangVersion=2.0.0-beta.2-20210607-150900-f557ec7f
 testngVersion=7.3.0
 slf4jVersion=1.7.30
 org.gradle.jvmargs=-Xmx4096M
 
 #stdlib
-stdlibIoVersion=0.6.0-beta.1
-stdlibRegexVersion=0.7.0-beta.1
-stdlibFileVersion=0.7.0-beta.1
-stdlibOsVersion=0.8.0-beta.1
-stdlibLogVersion=1.1.0-beta.1
-stdlibMimeVersion=1.1.0-beta.1
-stdlibCryptoVersion=1.1.0-beta.1
-stdlibAuthVersion=1.1.0-beta.1
-stdlibHttpVersion=1.1.0-beta.1
-stdlibTimeVersion=2.0.0-beta.1
-stdlibTaskVersion=2.0.0-beta.1
-stdlibCacheVersion=2.1.0-beta.1
-stdlibJwtVersion=1.1.0-beta.1
-stdlibOAuth2Version=1.1.0-beta.1
-stdlibUuidVersion=0.10.0-beta.1
-stdlibUrlVersion=1.1.0-beta.1
-stdlibXmlDataVersion=1.1.0-beta.1
+stdlibIoVersion=0.6.0-beta.2-20210607-175600-59c6281
+stdlibRegexVersion=0.7.0-beta.2-20210607-180600-636fac1
+stdlibFileVersion=0.7.0-beta.2-20210607-190100-1442a86
+stdlibOsVersion=0.8.0-beta.2-20210607-185200-21f29ef
+stdlibLogVersion=1.1.0-beta.2-20210607-185500-0d001db
+stdlibMimeVersion=1.1.0-beta.2-20210607-190200-a9c60bd
+stdlibCryptoVersion=1.1.0-beta.2-20210607-180000-b388165
+stdlibAuthVersion=1.1.0-beta.2-20210607-191000-ee8fbc0
+stdlibHttpVersion=1.1.0-beta.2-20210607-194800-e5989af
+stdlibTimeVersion=2.0.0-beta.2-20210607-163500-af65ddf
+stdlibTaskVersion=2.0.0-beta.2-20210607-182100-6c33b70
+stdlibCacheVersion=2.1.0-beta.2-20210607-190100-3ec59a0
+stdlibJwtVersion=1.1.0-beta.2-20210607-191100-fbadb0f
+stdlibOAuth2Version=1.1.0-beta.2-20210607-191400-bac3e3e
+stdlibUuidVersion=0.10.0-beta.2-20210607-190200-059ae06
+stdlibUrlVersion=1.1.0-beta.2-20210607-180000-8ec0d39
+stdlibXmlDataVersion=1.1.0-beta.2-20210607-180600-dcf8c8a
```

Comment:
Seems this is not the standard way to refer to xmldata. Better update the key too

Reply:
I will update :)